metpy/plots/tests/test_skewt.py

```diff
@@ -77,3 +77,7 @@ def test_hodograph_units():
     hodo.plot(u, v)
     hodo.plot_colormapped(u, v, np.sqrt(u * u + v * v), cmap='Greys')
     return fig
+
+def test_hodograph_alone():
+    hodo = Hodograph()
```

Comment:
```
/home/travis/build/Unidata/MetPy/metpy/plots/tests/test_skewt.py:81:1: D103 Missing docstring in public function
```
Need to add a docstring for the test. Just something like:
`"""Test creating Hodograph without passing an Axes."""`

Also:
```
/home/travis/build/Unidata/MetPy/metpy/plots/tests/test_skewt.py:82:5: F841 local variable 'hodo' is assigned to but never used```

Either just call `Hodograph()` or do something with the return variable, like `hodo.add_grid()`. Up to you.